src/data/live/data-source.js

```diff
@@ -0,0 +1,56 @@
+// import { dataSource as baseLive } from '@apollosproject/data-connector-church-online';
+import RockApolloDataSource from '@apollosproject/rock-apollo-data-source';
+import ApollosConfig from '@apollosproject/config';
+
+const { CHURCH_ONLINE, ROCK_MAPPINGS } = ApollosConfig;
+
+export default class LiveStream extends RockApolloDataSource {
+  resource = 'LiveStream';
+
+  mediaUrls = () => {
+    return CHURCH_ONLINE.MEDIA_URLS;
+  };
+
+  webViewUrl = () => {
+    return CHURCH_ONLINE.WEB_VIEW_URL;
+  };
+
+  async getLiveStream() {
+    return {
+      isLive: () => this.getIsLive(),
```

Comment:
probably doesn't need to be a separate function call, just 

`isLive: (await this.post(LAVA_STUFF)) === "true"`

Reply:
Yeah yeah ... that's a good call.